### PR TITLE
Blogpost: An update on the New Architecture Rollout

### PR DESCRIPTION
**Pull request by cortinico**
Time for a short update from the React Native team 🎉

### CODE REVIEW

File: website/blog/2022-03-14-an-update-on-the-new-architecture-rollout.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+If you’re a **library developer**, [we invite you to post an update](https://github.com/reactwg/react-native-new-architecture/discussions/categories/libraries) in the New Architecture working group with the **status of your libraries**. This will help you attract early adopters and us to understand if any library is facing a blocker.
+
+If instead you’re a **library user**, you can [post a message here](https://github.com/reactwg/react-native-new-architecture/discussions/6) to request a migration of a library. If we identify a library that becomes a blocker for a number of users, we will try to reach out to the maintainer and understand why they haven’t migrated yet.
+
+Finally, we would like to give a shout out to Software Mansion for releasing a new version of `[react-native-screens](https://github.com/software-mansion/react-native-screens)`, which has support for both architectures. Moreover, they published a blog-post ([Introducing Fabric to react-native-screens](https://blog.swmansion.com/introducing-fabric-to-react-native-screens-fd17bf18858e)) where they **tell their migration story**. We hope you will find this story inspiring and useful to tackle your migration.
```
Comment (kelset) [inline website/blog/2022-03-14-an-update-on-the-new-architecture-rollout.md:78]:
```suggestion
Finally, we would like to give a shout out to Software Mansion for releasing a new version of [`react-native-screens`](https://github.com/software-mansion/react-native-screens), which has support for both architectures. Moreover, they published a blog-post ([Introducing Fabric to react-native-screens](https://blog.swmansion.com/introducing-fabric-to-react-native-screens-fd17bf18858e)) where they **tell their migration story**. We hope you will find this story inspiring and useful to tackle your migration.
```
putting the `` the right place ;)

File: website/blog/2022-03-14-an-update-on-the-new-architecture-rollout.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+### Releases
+
+Work on the 0.68 pre-release has realized much of [the improved release process we had defined last half](/blog/2022/01/19/version-067#improvements-to-release-process).
+
+We’re happy to share that with 0.68 we were able to:
+
+- Successfully onboard release work to an internal rotation. Much of this is supported by [improved documentation](https://github.com/facebook/react-native/wiki/releases) on the release process which will reduce the bus factor of the release process.
+- Initiated discussions with partners to support a [co-pilot rotation](https://github.com/facebook/react-native/wiki/Release-Roles-and-Responsibilities#release-role-2--release-copilot). We hope this effort will improve transparency of the process and inform our partners where to invest to support React Native releases and eco-system.
```
Comment (Simek) [inline website/blog/2022-03-14-an-update-on-the-new-architecture-rollout.md:87]:
```suggestion
- Initiated discussions with partners to support a [co-pilot rotation](/contributing/release-roles-responsibilities#release-role-2--release-copilot). We hope this effort will improve transparency of the process and inform our partners where to invest to support React Native releases and eco-system.
```

Wiki pages lately has been moved into website:
* https://reactnative.dev/contributing/release-roles-responsibilities#release-role-2--release-copilot

File: website/blog/2022-03-14-an-update-on-the-new-architecture-rollout.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+As every discussion forum, we would like to stress once more the importance of being **respectful** and welcoming towards others’ opinions. Please take the chance to read our [**code of conduct**](https://github.com/reactwg/react-native-new-architecture/blob/main/CODE_OF_CONDUCT.md) if you haven’t yet done it.
+
+### The Migration Guide
+
+After several rounds of review & feedback, we finally merged **the Migration Guide** (f.k.a. _the Playbook_). You can find it [on the website in the Guides section.](/docs/next/new-architecture-intro)
+
+This Migration Guide will show you **how to create a custom Fabric component or a TurboModule** with a step-by-step approach. The guide will also show you how to **adapt your existing app or library** to use the New Architecture.
+
+Moreover, we would like to remind you the brand-new [Architecture section](/docs/next/architecture-overview) of our website. There you can find several in-depth articles and explanation of the React Native internals. Specifically, [the Fabric section](/docs/next/fabric-renderer) can help you understand the rendering pipeline in the New Architecture world.
```
Comment (Simek) [inline website/blog/2022-03-14-an-update-on-the-new-architecture-rollout.md:40]:
```suggestion
Moreover, we would like to remind you the brand-new [Architecture section](/architecture/overview) of our website. There you can find several in-depth articles and explanation of the React Native internals. Specifically, [the Fabric section](/architecture/fabric-renderer) can help you understand the rendering pipeline in the New Architecture world.
```

Links have changed after latest conversion of the architecture section page to unversioned docs.

File: website/blog/2022-03-14-an-update-on-the-new-architecture-rollout.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
+---
+title: An update on the New Architecture Rollout
+authors: [cortinico]
+tags: [announcement]
```
Comment (Simek) [inline website/blog/2022-03-14-an-update-on-the-new-architecture-rollout.md:4]:
```suggestion
tags: [announcement]
date: 2022-03-15
```

Probably the date should be adjusted before publishing:
https://docusaurus.io/docs/next/api/plugins/@docusaurus/plugin-content-blog#date

File: website/blog/2022-03-14-an-update-on-the-new-architecture-rollout.md [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+  - Set an environment variable `ORG_GRADLE_PROJECT_newArchEnabled=true`
+  - Invoke Gradle with `-PnewArchEnabled=true`
+
+Then you can **run your app** with `yarn react-native run-android` or `run-ios` and you’ll be running using Fabric and TurboModules enabled.
+
+Please consider trying this new template, and [report any bug or unexpected behavior](https://github.com/reactwg/react-native-new-architecture/discussions/5) that you might face. Over the last months we worked hard to fix bugs and build failures that would have been **hard to catch** without the constant community feedback and testing.
+
+### The 3rd-party Libraries Ecosystem
+
+We could not migrate to the New Architecture without the full support of **3rd-party libraries author & maintainers**.
```
Comment (kelset) [inline website/blog/2022-03-14-an-update-on-the-new-architecture-rollout.md:70]:
```suggestion
We can not migrate to the New Architecture without the full support of **3rd-party libraries author & maintainers**.
```
nit: this should not be using the past for "could", right? It's not like the new arch is out and everyone has migrated to it.

Or maybe it should be even more clear, along the lines of:
> The community won't be able to migrate to the New Architecture without the support of their fellow **libraries author & maintainers**.

File: website/blog/2022-03-14-an-update-on-the-new-architecture-rollout.md [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+
+We’re happy to share that with 0.68 we were able to:
+
+- Successfully onboard release work to an internal rotation. Much of this is supported by [improved documentation](https://github.com/facebook/react-native/wiki/releases) on the release process which will reduce the bus factor of the release process.
+- Initiated discussions with partners to support a [Copilot rotation](/contributing/release-roles-responsibilities#release-role-2--release-copilot). We hope this effort will improve transparency of the process and inform our partners where to invest to support React Native releases and eco-system.
+- [Onboarded several Release Supporters and Testers from the community](https://github.com/reactwg/react-native-releases/discussions/11). We had put a call-out for help last half and so many folks stepped up! The feedback from our testers and supporters have **helped us fix crucial bugs** and regressions, especially around the new architecture, for the upcoming release. Thank you to everyone who signed up and tested out the release! If you’d like to join as a release tester or supporter, please sign up here!
```
Comment (kelset) [inline website/blog/2022-03-14-an-update-on-the-new-architecture-rollout.md:88]:
```suggestion
- [Onboarded several Release Supporters and Testers from the community](https://github.com/reactwg/react-native-releases/discussions/11). We had put a call-out for help last half and so many folks stepped up! The feedback from our testers and supporters have **helped us fix crucial bugs** and regressions, especially around the new architecture, for the upcoming release. Thank you to everyone who signed up and tested out the release! If you’d like to join as a release tester or supporter, [please sign up here](ADD ME)!
```
Link is missing

File: website/blog/2022-03-14-an-update-on-the-new-architecture-rollout.md [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+
+Finally, we would like to give a shout out to Software Mansion for releasing a new version of [`react-native-screens`](https://github.com/software-mansion/react-native-screens), which has support for both architectures. Moreover, they published a blog-post ([Introducing Fabric to react-native-screens](https://blog.swmansion.com/introducing-fabric-to-react-native-screens-fd17bf18858e)) where they **tell their migration story**. We hope you will find this story inspiring and useful to tackle your migration.
+
+### Releases
+
+Work on the 0.68 pre-release has realized much of [the improved release process we had defined last half](/blog/2022/01/19/version-067#improvements-to-release-process).
+
+We’re happy to share that with 0.68 we were able to:
+
+- Successfully onboard release work to an internal rotation. Much of this is supported by [improved documentation](https://github.com/facebook/react-native/wiki/releases) on the release process which will reduce the bus factor of the release process.
```
Comment (Simek) [inline website/blog/2022-03-14-an-update-on-the-new-architecture-rollout.md:86]:
```suggestion
- Successfully onboard release work to an internal rotation. Much of this is supported by [improved documentation](/contributing/how-to-contribute) on the release process which will reduce the bus factor of the release process.
```

One more link updated, after wiki content move.

**Review by Simek — APPROVED** [review]
Just two more small suggestions, I have missed the one wiki link earlier, other than that it LGTM! 👍

File: website/blog/2022-03-14-an-update-on-the-new-architecture-rollout.md (hunk 6 above)
Comment (cortinico) [reply website/blog/2022-03-14-an-update-on-the-new-architecture-rollout.md]: Removing for now as the sign up is on hold for now (cc @lunaleaps).

File: website/blog/2022-03-14-an-update-on-the-new-architecture-rollout.md (hunk 5 above)
Comment (cortinico) [reply website/blog/2022-03-14-an-update-on-the-new-architecture-rollout.md]: Fixed 👍

File: website/blog/2022-03-14-an-update-on-the-new-architecture-rollout.md (hunk 6 above)
Comment (lunaleaps) [reply website/blog/2022-03-14-an-update-on-the-new-architecture-rollout.md]: https://forms.gle/fPuPE1MZRDGWNqpd6

File: website/blog/2022-03-14-an-update-on-the-new-architecture-rollout.md [hunk 8]
```diff
@@ -84,7 +84,7 @@ Work on the 0.68 pre-release has realized much of [the improved release process
...
-- Initiated discussions with partners to support a [co-pilot rotation](https://github.com/facebook/react-native/wiki/Release-Roles-and-Responsibilities#release-role-2--release-copilot). We hope this effort will improve transparency of the process and inform our partners where to invest to support React Native releases and eco-system.
+- Initiated discussions with partners to support a [Copilot rotation](/contributing/release-roles-responsibilities#release-role-2--release-copilot). We hope this effort will improve transparency of the process and inform our partners where to invest to support React Native releases and eco-system.
 - [Onboarded several Release Supporters and Testers from the community](https://github.com/reactwg/react-native-releases/discussions/11). We had put a call-out for help last half and so many folks stepped up! The feedback from our testers and supporters have **helped us fix crucial bugs** and regressions, especially around the new architecture, for the upcoming release. Thank you to everyone who signed up and tested out the release! If you’d like to join as a release tester or supporter, please sign up here!
```
Comment (lunaleaps) [inline website/blog/2022-03-14-an-update-on-the-new-architecture-rollout.md]:
```suggestion
- [Onboarded several Release Supporters and Testers from the community](https://github.com/reactwg/react-native-releases/discussions/11). We had put a call-out for help last half and so many folks stepped up! The feedback from our testers and supporters have **helped us fix crucial bugs** and regressions, especially around the new architecture, for the upcoming release. Thank you to everyone who signed up and tested out the release! If you’d like to join as a release tester or supporter, [please sign up here](https://forms.gle/fPuPE1MZRDGWNqpd6)!
```

**Pull request merged**